qa/backwards-5.0/build.gradle

```diff
@@ -16,9 +16,9 @@ apply plugin: 'elasticsearch.rest-test'
 integTest {
   includePackaged = true
   cluster {
-    numNodes = 2
-    numBwcNodes = 1
-    bwcVersion = "6.0.0-alpha1-SNAPSHOT"
+    numNodes = 4
+    numBwcNodes = 2
```

Comment:
Yeah, I was doubting on that one too. 4 nodes total is really required to have a good bwc case for the global checkpoint management (we need two replicas and one primary). I see what you are saying but I was doubting whether it's worth introducing a whole now BWC cluster. Setting it up might be slower than just running the yaml tests..